### PR TITLE
chore: implement make target for multi arch image build

### PR DESCRIPTION
**Pull request by daixiang0**
Fix https://github.com/envoyproxy/gateway/issues/138

Signed-off-by: Loong Dai <loong.dai@intel.com>

### CODE REVIEW

File: tools/make/image.mk [hunk 1]
```diff
@@ -67,6 +64,27 @@ image.push.%:
...
+.PHONY: image.multiarch.verify
+image.multiarch.verify:
+	$(eval pass := $(shell ))
+	@docker buildx --help | grep -qw "docker buildx" || { \
+		echo "Cannot find `docker buildx`, please install first"; \
+		exit 1; }
+
+.PHONY: image.multiarch.setup
+image.multiarch.setup: image.verify image.multiarch.verify
+	@docker run --rm --privileged tonistiigi/binfmt --install all # Install QEMU emulators
```
Comment (arkodg) [inline tools/make/image.mk:76]: it is safer to only install emulators for the interested platforms `--platform "${BUILDX_PLATFORMS}`

Comment (daixiang0) [reply tools/make/image.mk:76]: Sure, let me fix it.

File: tools/make/image.mk [hunk 2]
```diff
@@ -67,6 +64,27 @@ image.push.%:
...
 
+.PHONY: image.multiarch.verify
+image.multiarch.verify:
+	$(eval pass := $(shell ))
+	@docker buildx --help | grep -qw "docker buildx" || { \
+		echo "Cannot find `docker buildx`, please install first"; \
+		exit 1; }
+
+.PHONY: image.multiarch.setup
+image.multiarch.setup: image.verify image.multiarch.verify
```
[the remark this answers is not in the document]
Comment (daixiang0) [reply tools/make/image.mk]: Add it.

File: tools/make/image.mk [hunk 3]
```diff
@@ -67,6 +66,32 @@ image.push.%:
...
+		exit 1; }
+
+.PHONY: image.multiarch.emulate $(EMULATE_TARGETS)
+image.multiarch.emulate: $(EMULATE_TARGETS)
+$(EMULATE_TARGETS): image.multiarch.emulate.%:
+	@docker run --rm --privileged tonistiigi/binfmt --install linux/$* # Install QEMU emulator, the same emulator as the host will report an error but can safe ignore
+
+.PHONY: image.multiarch.setup
+image.multiarch.setup: image.verify image.multiarch.verify image.multiarch.emulate
+	@docker run --rm --privileged tonistiigi/binfmt --install all # Install QEMU emulators
```
Comment (arkodg) [inline tools/make/image.mk:83]: shouldnt this line be deleted @daixiang0 ?

Comment (daixiang0) [reply tools/make/image.mk:83]: I miss it, post anther PR to fix it.

**Pull request merged**